segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/filesystem/FileSystemStorage.java

```diff
@@ -409,7 +409,7 @@ private Throwable translateException(String segmentName, Throwable e) {
             retVal = new IllegalArgumentException(e.getMessage());
         }
 
-        if (e instanceof AccessDeniedException) {
+        if (e instanceof AccessControlException) {
```

Comment:
I'm not sure this is correct either, if we check the code for openjdk, there is no `AccessControlException` being thrown:

http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/8u40-b25/sun/nio/fs/UnixException.java?av=f#77